docs/docs/metrics/write-your-own.md

```diff
@@ -24,74 +24,85 @@ You can use the following template to get started with writing your own metric.
 Your implementation should call `writer.write(<object_score>, <object>)` for every object in the iterator or use `writer.write(<frame_score>)` for every data unit in the iterator.
```

Comment:
```suggestion
Your implementation should call `writer.write(<object_score>, <object>)` for every object in the iterator **OR** use `writer.write(<frame_score>)` for every data unit in the iterator.
```

Reply:
Perhaps add a line saying (for now) that one should not do both in the same metric definition.